### PR TITLE
Fix issue 180. WrongUsageOfMappersFactory componentModel from config

### PR DESCRIPTION
**Pull request by hduelme**
I added support to read the `componentModel` from a `MapperConfig` for `WrongUsageOfMappersFactoryInspection` see #180 .
I only added the quickfix to remove `Mappers` usage. I think it is more intuitive to not write to other files for this. If we want to support this we must ensure that we could write to the `MapperConfig` (could be provided as a dependency for example).

### CODE REVIEW

File: src/main/java/org/mapstruct/intellij/inspection/WrongUsageOfMappersFactoryInspection.java [hunk 1]
```diff
@@ -45,6 +46,7 @@ public class WrongUsageOfMappersFactoryInspection extends InspectionBase {
         MapstructUtil.MAPPERS_FQN,
         "getMapper"
     ).parameterTypes( CommonClassNames.JAVA_LANG_CLASS );
+    public static final String COMPONENT_MODEL = "componentModel";
```
Comment (filiphr) [inline src/main/java/org/mapstruct/intellij/inspection/WrongUsageOfMappersFactoryInspection.java:49]: Does this need to be `public`?

**Review by filiphr — COMMENTED** [review]
Great catch @hduelme. Only one small comment

File: src/main/java/org/mapstruct/intellij/inspection/WrongUsageOfMappersFactoryInspection.java (hunk 1 above)
Comment (hduelme) [reply src/main/java/org/mapstruct/intellij/inspection/WrongUsageOfMappersFactoryInspection.java]: No doesn't need to be. Change to `private`.

**Comment by hduelme** [comment]
@filiphr Changed

**Pull request merged**